site/content/docs/tutorials/concierge-and-supervisor-gke-demo.md

```diff
@@ -0,0 +1,462 @@
+---
+title: Use the Pinniped Supervisor and Concierge for federated login on GKE
```

Comment:
Can we frame this as a getting started guide?  This title already assumes you know what the supervisor and concierge is.  Something like "Getting started with Pinniped, from zero to hero" or "Pinniped the hard way."  This would then be prominently displayed on pinniped.dev and pinniped.dev/docs

Reply:
Made several prominent links to the new doc, as we discussed offline. The new doc will become the primary tutorial on our site.